### PR TITLE
Use AAA pattern

### PR DESCRIPTION
**Pull request by mafeZapata**

### CODE REVIEW

**Review by holgiosalos — COMMENTED** [review]
I only see changes over the package.json and package-lock.json files. More changes are expected for this section

File: package.json [hunk 1]
```diff
@@ -7,7 +7,8 @@
     "test:open": "npm run lint && cypress open",
     "lint": "eslint ./cypress/e2e/**/*.ts",
     "lint:fix": "npm run lint -- --fix",
-    "test": "cypress run"
+    "test": "cypress open"
```
Comment (manuelq12) [inline package.json]: Why did you change this?
Remember is better to have this script as headless for the CI

As well, i don't see the changes from the cy.ts files

**Review by manuelq12 — CHANGES_REQUESTED** [review]

File: cypress/e2e/buy-shirt.cy.ts [hunk 2]
```diff
@@ -17,6 +17,28 @@ const addressStepPage = new AddressStepPage();
 const shippingStepPage = new ShippingStepPage();
 const paymentStepPage = new PaymentStepPage();
 
+<<<<<<< Updated upstream
+=======
```
Comment (manuelq12) [inline cypress/e2e/buy-shirt.cy.ts]: Can you erase this lines please

File: cypress/e2e/buy-shirt.cy.ts [hunk 3]
```diff
@@ -17,6 +17,28 @@ const addressStepPage = new AddressStepPage();
...
+    menuContentPage = new MenuContentPage();
+    productsListPage = new ProductsListPage();
+    shoppingCartPage = new ShoppingCartPage();
+    loginPage = new LoginPage();
+    addressStepPage = new AddressStepPage();
+    shippingStepPage = new ShippingStepPage();
+    paymentStepPage = new PaymentStepPage();
+    menuContentPage.visitMenuContentPage();
+  });
+>>>>>>> Stashed changes
```
Comment (manuelq12) [inline cypress/e2e/buy-shirt.cy.ts]: This one

File: cypress/e2e/buy-shirt.cy.ts (hunk 2 above)
Comment (holgiosalos) [inline cypress/e2e/buy-shirt.cy.ts]: remove these lines

File: cypress/e2e/buy-shirt.cy.ts (hunk 3 above)
Comment (holgiosalos) [inline cypress/e2e/buy-shirt.cy.ts]: remove this line

File: cypress/page/menu-content.page.ts [hunk 4]
```diff
@@ -1,6 +1,11 @@
 class MenuContentPage {
+    private dressesMenu: string;
     private tShirtMenu: string;
     private menuContentPageURL: string
+<<<<<<< Updated upstream
+=======
+    
+>>>>>>> Stashed changes
```
Comment (manuelq12) [inline cypress/page/menu-content.page.ts]: These 4 also

**Review by manuelq12 — CHANGES_REQUESTED** [review]

File: cypress/page/menu-content.page.ts (hunk 4 above)
Comment (holgiosalos) [inline cypress/page/menu-content.page.ts]: remove these lines

**Review by holgiosalos — COMMENTED** [review]
remember to run the lint:fix command before pushing

File: cypress/page/menu-content.page.ts [hunk 5]
```diff
@@ -4,9 +4,10 @@ class MenuContentPage {
     private dressesMenu: string;
 
     constructor() {
-        this.tShirtMenu = "#block_top_menu > ul > li > a[title='T-shirts']";
+        this.tShirtMenu = "#block_top_menu > .sf-menu > li > a[title='T-shirts']";
         this.dressesMenu = "#block_top_menu > .sf-menu > li > a[title='Dresses']";
```
Comment (holgiosalos) [inline cypress/page/menu-content.page.ts]: I would use the `ul` instead of `.sf-menu`

File: package.json (hunk 1 above)
Comment (holgiosalos) [reply package.json]: yep, it should be cypress run intead of cypress open

**Review by holgiosalos — APPROVED** [review]

**Review by manuelq12 — APPROVED** [review]

**Pull request merged**